### ISSUE TITLE
Missing or wrongly related labels and headings

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wcag-em-report-tool#356]
_Passed on from an e-mail to wai@w3.org:_
I tried it with MS Edge/JAWS and with Chrome/NVDA, but it was in both cases difficult to get labels for text fields and dropdown lists and to read text entered into the observation text fields. Furthermore, I would appreciate if the SC were marked as headings and if the sample pages in the audit screen could get tab focus in order to make sure that keyboard users like me find them in their reading order between two focusable elements (audit observation text field and audit result dropdown).

**Comment by rvantonisse** [comment]
# Mentioned issues found
> I tried it with MS Edge/JAWS and with Chrome/NVDA, but it was in both cases difficult to get labels for text fields and dropdown lists and to read text entered into the observation text fields.

This is not browser/screenreader related, but related to the implementation of code markup. Some things are going wrong and this needs to be fixed! Valuable input!

## **text inputs** without a related label (no name / missing relation)
- [ ] Step: Sample > Structured and random sample text input labelling is mixed up
- [ ] Step: Audit > Observations text input has missing translations and reads `AUDIT.LABEL_DESCR` to screenreaders (This is fixed, see [preview audit step](https://w3c.github.io/wcag-em-report-tool/dist/#!/evaluation/audit), but needs a new deploy to the live version)

## **select / combobox** inputs without a related label (no name / missing relation)

- [ ] Step Scope > Wcag version label has wrong for value: is `version_target` should be `wcag_version`
- [ ] Step audit > Observation result (outcome) select has general label "outcome" and context is not clear. This can be fixed adding a fieldset around the criterion and adding "Results for the entire sample" / "Results for ...url..." as label. This would give: "Outcome Results for …" and "Observation Results for …" for the textfield to the screenreader.

## Audit step Succes criteria visually presented **titles** are not coded as html headings (missing info / relation)

This is something to give a better look. Headings should work, but another possibility is to change the complete criterion in a group with a legend containing the title since what is coming are form elements with a lot of the same labels. Kind of seeing a list of "Read more"-links. Fieldset + legend is a solution for this. 

## Make sample url text in observations a link (UI) so it is tabable
This is also requested in a separate issue to simply be able to open the url provided, see #304

**Comment by slhenry** [comment]
@nitedog fyi

**Comment by hidde** [comment]
We recently launched the [3.0.0 version of the WCAG-EM Report Tool](https://w3.org/WAI/eval/report-tool), in which this issue is fixed.
